api/overview/azure/cognitiveservices/face-readme.md

```diff
@@ -4,10 +4,10 @@ author: PatrickFarley
 ms.author: pafarley
 ms.date: 06/21/2021
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: csharp
-ms.service: face-api
+ms.service: cognitive-services
```

Comment:
This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv